pkgs/by-name/pi/pihole/package.nix

```diff
@@ -0,0 +1,259 @@
+{
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  installShellFiles,
+  bash,
+  coreutils,
+  curl,
+  dig,
+  gawk,
+  getent,
+  glibc,
+  gnugrep,
+  gnused,
+  iproute2,
+  jq,
+  killall,
+  libidn2,
+  locale,
+  ncurses,
+  netcat,
+  nettools,
+  pihole-ftl,
+  procps,
+  resholve,
+  sqlite,
+  systemd,
+  util-linux,
+  stateDir ? "/etc/pihole",
+  ...
+}:
+
+(resholve.mkDerivation rec {
```

Comment:
What is `resholve.mkDerivation`? Can we get rid of `rec` and use the `finalAttrs` pattern?